MachObfuscator/Mach/Parsing/Mach+selectors.swift

```diff
@@ -1,6 +1,31 @@
 import Foundation
 
 extension Mach {
+    // See `sel_init` function in https://opensource.apple.com/source/objc4/objc4-750.1/runtime/objc-sel.mm.auto.html
+    static var libobjcSelectors: [String] { return [
```

Comment:
Could you please just add the code comment that this list is important only in case of using the `xx-no-analyze-dependencies`? I will merge just right after that :)

Reply:
comment added